src/commands/reply.ts

```diff
@@ -54,6 +54,11 @@ async function run(
           "you must provide evidence the old account username and user ID, as well as prove that it is your account\n\n" +
           "if you're unable to prove that it's your account, we cannot do anything.",
       );
+    } else if (args[0].toLowerCase() === "auto.buyunban") {
+      embed.setDescription(
+        "if you are **banned/muted from the nypsi discord server** then you can be unbanned by making a custom donation of £20 to https://ko-fi.com/tekoh\n\n" +
```

Comment:
slight change of wording

```suggestion
        "if you are **banned/muted from the nypsi discord server** then you can be unbanned/unmuted by making a custom donation of £20 to https://ko-fi.com/tekoh\n\n" +
```